src/core/engines/BaseEngine.sol

```diff
@@ -176,12 +183,36 @@ abstract contract BaseEngine is ReentrancyGuard {
         optionToken.mint(recipient, tokenId, amount);
     }
 
+    /**
+     * @dev mint option token into account, increase short position (debt) and increase long position in account memory
+     * @param _data bytes data to decode
+     */
+    function _mintOptionIntoAccount(address _subAccount, bytes memory _data) internal virtual {
+        // decode parameters
+        (uint256 tokenId, address recipient, uint64 amount) = abi.decode(_data, (uint256, address, uint64));
```

Comment:
better rename `recipient` to  `recipientSubAccount`

Reply:
Also update `createMintIntoAccountAction`